packages/api/src/user.js

```diff
@@ -159,23 +159,23 @@ export function withAuth (handler) {
 export async function userTokensPost (request, env) {
   const { name } = await request.json()
   if (!name || typeof name !== 'string') {
-    throw new Error('invalid name')
+    throw Object.assign(new Error('invalid name'), { status: 400 })
   }
 
   const { _id, issuer } = request.auth.user
   const sub = issuer
   const iss = JWT_ISSUER
   const secret = await JWT.sign({ sub, iss, iat: Date.now(), name }, env.SALT)
 
-  await env.db.query(gql`
+  const res = await env.db.query(gql`
     mutation CreateAuthToken($data: CreateAuthTokenInput!) {
       createAuthToken(data: $data) {
         _id
       }
     }
   `, { data: { user: _id, name, secret } })
 
-  return new JSONResponse()
+  return new JSONResponse(res.createAuthToken, { status: 201 })
```

Comment:
Is this the pattern we want? To always return the auth token?

Reply:
Just selecting the ID from the query and returning it.